plugins/nv-filters/nvidia-videofx-filter.c

```diff
@@ -245,36 +246,47 @@ static bool nvvfx_filter_create_internal(struct nvvfx_data *filter)
 		log_nverror_destroy(filter, vfxErr);
 	/* debug */
 #ifdef _DEBUG
-	const char *info;
-	vfxErr = NvVFX_GetString(filter->handle_blur, NVVFX_INFO, &info);
-	blog(LOG_DEBUG, "blur fx settings /n/%s", info);
+	if (id == S_FX_AIGS || id == S_FX_BG_BLUR) {
+		const char *info2;
+		vfxErr = NvVFX_GetString(filter->handle, NVVFX_INFO, &info2);
+		blog(LOG_DEBUG, "aigs fx settings :%s", info2);
+	}
+	if (id == S_FX_BLUR || id == S_FX_BG_BLUR) {
+		const char *info;
+		vfxErr = NvVFX_GetString(filter->handle_blur, NVVFX_INFO, &info);
+		blog(LOG_DEBUG, "blur fx settings :%s", info);
+	}
 #endif
 	/* 2. Set models path & initialize CudaStream */
+	vfxErr = NvVFX_CudaStreamCreate(&filter->stream);
+	if (NVCV_SUCCESS != vfxErr)
+		log_nverror_destroy(filter, vfxErr);
```

Comment:
Leave a blank line after unenclosed `if` statements, or enclose the `if` with curly braces.